qualtran/simulation/classical_sim.py

```diff
@@ -81,16 +81,23 @@ def _numpy_dtype_from_qdtype(dtype: 'QDType') -> Type:
     return object
 
 
+def _numpy_empty_reg_from_reg(reg: Register) -> np.ndarray:
```

Comment:
"nump" and "from reg" makes sense; but we're not returning an empty register per se. `_numpy_empty_array_from_reg`? `_np_empty_from_reg`? `_empty_ndarray_from_reg`?

Reply:
Should this function be an overridable method on QDType? Not blocking this PR but if you think it is a good idea we can open an issue

Reply:
Yes, I think it's a good idea to have this function on then QDType. It would also make `Fxp` integration better I think; since `Fxp` also derives from numpy array and we don't make use of that. This is also why `Fxp` is slow for us I think. 

I'll open an issue.

Reply:
Did you end up opening one? There's something during review of #1436 that I'd like to note on the issue as well

Reply:
Opened https://github.com/quantumlib/Qualtran/issues/1437